.circleci/config.base.yml

```diff
@@ -97,6 +97,8 @@ jobs:
       - run:
           name: Collect code coverage
           command: yarn coverage
+    environment:
+      NODE_OPTIONS: --max-old-space-size=4096
```

Comment:
Because of Because of https://app.circleci.com/pipelines/github/aws-amplify/amplify-category-api/1548/workflows/86e8aeb5-342e-4526-9370-2b7789e24d18/jobs/40319 .
I .

Borrowed from https://github.com/aws-amplify/amplify-cli/blob/015187a8b714b96fbb87331383a92664aa6e6f35/.circleci/config.base.yml#L134-L148